gems/pending/spec/middleware/events/middleware_event_monitor_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "middleware/events/middleware_event_monitor"
+
+describe MiddlewareEventMonitor do
+  subject do
+    allow(MiqServer).to receive(:my_zone).and_return("default")
+    auth = AuthToken.new(:name => "test", :auth_key => "valid-token", :userid => "jdoe", :password => "password")
+    ems  = FactoryGirl.create(:ems_hawkular,
+                              :hostname        => 'localhost',
+                              :port            => 8080,
+                              :authentications => [auth])
+    WebMock.allow_net_connect!
```

Comment:
Ah, ok. The cassette will be generated if it does not exist. So to regenerate one, just remove the vcr yml file and re run the test